cli-toolchain/graphql.md

```diff
@@ -914,255 +913,103 @@ mutation CreateDraft {
-			"Description": "The id of the AppSync API associated with this project."
-		},
-		"S3DeploymentBucket": {
-			"Type": "String",
-			"Description": "The S3 bucket containing all deployment assets for the project."
-		},
-		"S3DeploymentRootKey": {
-			"Type": "String",
-			"Description": "An S3 key relative to the S3DeploymentBucket that points to the root\nof the deployment directory."
-		}
-	},
-	"Resources": {
-		"SubscriptionOnCreateTodo": {
-			"Type": "AWS::AppSync::Resolver",
-			"Properties": {
-				"ApiId": {
-                    "Ref": "AppSyncApiId"
-                },
-                "DataSourceName": "Local",
-                "FieldName": "onCreateSalary",
-                "TypeName": "Subscription",
-                "RequestMappingTemplateS3Location": {
-                    "Fn::Sub": [
-                        "s3://${S3DeploymentBucket}/${S3DeploymentRootKey}/resolvers/${ResolverFileName}",
-                        {
-                            "S3DeploymentBucket": {
-                                "Ref": "S3DeploymentBucket"
-                            },
-                            "S3DeploymentRootKey": {
-                                "Ref": "S3DeploymentRootKey"
-                            },
-                            "ResolverFileName": "Subscription.onCreateSalary.req.vtl"
-                        }
-                    ]
-                },
-                "ResponseMappingTemplateS3Location": {
-                    "Fn::Sub": [
-                        "s3://${S3DeploymentBucket}/${S3DeploymentRootKey}/resolvers/${ResolverFileName}",
-                        {
-                            "S3DeploymentBucket": {
-                                "Ref": "S3DeploymentBucket"
-                            },
-                            "S3DeploymentRootKey": {
-                                "Ref": "S3DeploymentRootKey"
-                            },
-                            "ResolverFileName": "Subscription.onCreateSalary.res.vtl"
-                        }
-                    ]
-                }
-			}
-		},
-		"TodoDataSource": {
-            "Type": "AWS::AppSync::DataSource",
-            "Properties": {
-                "ApiId": {
-                    "Ref": "AppSyncApiId"
-                },
-                "Name": "Local",
-                "Type": "NONE"
-            }
-        }
-	}
-}
-```
+Finally, if you use both owner and group authorization then the username argument becomes optional. This means the following:
 
-Lastly, add the resolver mapping templates to the API project's `resolvers/` directory.
+- If you don’t pass the user in, but are a member of an allowed group, the subscription will notify you of records added.
+- If you don’t pass the user in, but are NOT a member of an allowed group, the subscription will fail to connect.
+- If you pass the user in who IS the owner but is NOT a member of a group, the subscription will will notify you of records added of which you are the owner.
```

Comment:
_typo (double 'will'):_ If you pass the user in who IS the owner but is NOT a member of a group, the subscription will **will** notify you of records added of which you are the owner

Reply:
Fixed!